threadpoolctl.py

```diff
@@ -262,6 +276,9 @@ def _get_version(dynlib, internal_api):
         return _get_openblas_version(dynlib)
     elif internal_api == "blis":
         return _get_blis_version(dynlib)
+    elif internal_api == "tbb":
+        # tbb does not expose it's version
```

Comment:
hmm.. that's an interesting question. adding an issue

Reply:
Actually we need the version of libtbb DSO. So I guess TBB_runtime_interface_version() is what we want. Is that right ?
Thanks

Reply:
Yes, I believe so.